builder/cloudstack/config.go

```diff
@@ -34,6 +34,7 @@ type Config struct {
 	Hypervisor        string   `mapstructure:"hypervisor"`
 	InstanceName      string   `mapstructure:"instance_name"`
 	Keypair           string   `mapstructure:"keypair"`
+	TemporaryKeypair  string   `mapstructure:"temporary_keypair"`
```

Comment:
Yes I was hesitating about naming it `temporary_keypair` or `temporary_keypair_name`. I'll change, the later is better.